fix: Disable next/prev buttons in land page pagination #1011

Chakravarthy7102 opened the pull request:
Closes #1011

LautaroPetaccio commented:
Hi @Chakravarthy7102 ! Thanks again for your contribution, your code looks great.
What you did fixes the same issue described in the linked issue but for the pool of scenes. To solve the issue, the same fix should be applied in `src/components/ScenesPage/ScenesPage.tsx`, would you mind fixing it there as well?
I've also updated the PR description to include "Closes https://github.com/decentraland/builder/issues/1011" which is the phrase Github uses to link the PR with the issue.

Chakravarthy7102 commented:
> Hi @Chakravarthy7102 ! Thanks again for your contribution, your code looks great. What you did fixes the same issue described in the linked issue but for the pool of scenes. To solve the issue, the same fix should be applied in `src/components/ScenesPage/ScenesPage.tsx`, would you mind fixing it there as well? I've also updated the PR description to include "Closes #1011" which is the phrase Github uses to link the PR with the issue.

Fixed pagination in `src/components/ScenesPage/ScenesPage.tsx`
